test/ClaimsSpec.hs

```diff
@@ -4,20 +4,35 @@ import           Protolude
 
 import qualified Data.HashMap.Strict as M
 import           Test.Hspec
-import           Data.Aeson          (Value (..) )
+import           Data.Aeson          (Value (..),  toJSON, Array, fromJSON )
 import Data.Time.Clock
 import           PostgresWebsockets.Claims
 
+secret :: ByteString
+secret = "reallyreallyreallyreallyverysafe"
+-- it would be nice if we could use JOSE? to generate the test tokens...
```

Comment:
I would rather not generate test data dynamically unless we are doing some property testing